framework/base/ErrorException.php

```diff
@@ -17,6 +17,8 @@
  */
 class ErrorException extends \ErrorException
 {
+    const E_HHVM_FATAL_ERROR = 16777217;
```

Comment:
where is this constant from, it does not seem to be documented: http://docs.hhvm.com/manual/en/errorfunc.constants.php